src/Amalgam/Concurrency.h

```diff
@@ -82,6 +82,7 @@ namespace Concurrency
 
 #ifdef MULTITHREAD_SUPPORT
 	extern ThreadPool threadPool;
+	extern ThreadPool urgentThreadPool;
```

Comment:
I think a comment here on the usage between the 2 would be a good idea. Something to give a at least some guidance on when/how to use one or the other.